d3/hyper.js

```diff
@@ -30,159 +30,118 @@ Miscellaneous
 
 var common = require('./common');
 
+// Convert to string. Return undefined if not supported.
```

Comment:
kind of a Javascript question, does this function return undefined if no explicit return statement was called?

Reply:
yes, `undefined` is the default return value if no value is returned.
Probably I should explicitly write `return undefined, though

Reply:
modified the pull request to include `return undefined`